WalletWasabi.Fluent/ViewModels/Settings/BitcoinTabSettingsViewModel.cs

```diff
@@ -115,5 +118,22 @@ protected override void EditConfigOnSave(Config config)
 				BitcoinP2PEndPoint = config.GetBitcoinP2pEndPoint().ToString(defaultPort: -1);
 			}
 		}
+
+		public static async Task<string> GetNodeVersionAsync()
+		{
+			var version = "";
+			try
+			{
+				using var node = await Node.ConnectAsync(Services.Config.Network, Services.Config.GetBitcoinP2pEndPoint());
+				node.VersionHandshake();
+				version = node.PeerVersion.UserAgent;
+				Logger.LogDebug($"Full Node Version: {version} ");
+			}
+			catch (Exception ex)
+			{
+				Logger.LogDebug(ex);
```

Comment:
```suggestion
				Logger.LogError(ex);
```
IMO we want to see in the logs, when we couldn't even get our Full Node's version.

Reply:
Done